cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml

```diff
@@ -0,0 +1,45 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/latest/onlineEndpoint.schema.json
+name: my-endpoint
```

Comment:
This's not a must and it won't work as we specify the endpoint name in az cli command.
I prefer deleting name setting.

Reply:
name setting has been removed